### PR TITLE
SYS_MC_EST_GROUP mark LPE unsupported and update airframes

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

**Comment by dagar** [comment]
> Though, we should try to focus out efforts in making the platform estimator-agnostic sonner or later.

I agree, we should have interfaces for these things to enable maintainers to keep on top of what's required.

Other than lack of testing what really pushed this over the edge was knowing things like the bias corrected acceleration publication (used by the land detector) are missing in LPE.

I also want to prevent EKF2 from becoming a monolith. Things like GPS blending or the sensor offset parameters should all be outside of the estimator frontend.

**Comment by LorenzMeier** [comment]
Build failure points towards ROS timing issue: ```[ INFO] [1556493472.805969178]: Finished loading Gazebo ROS API Plugin.
[ INFO] [1556493472.813863390]: waitForService: Service [/gazebo/set_physics_properties] has not been advertised, waiting...
[ INFO] [1556493474.584761595, 0.024000000]: waitForService: Service [/gazebo/set_physics_properties] is now available.
[ INFO] [1556493474.658946484, 0.092000000]: Physics dynamic reconfigure ready.
... logging to /tmp/jenkins/workspace/PX4_Firmware_PR-11925/.ros/log/rostest-7f88920af52c-12113.log
[ROSUNIT] Outputting test results to /tmp/jenkins/workspace/PX4_Firmware_PR-11925/.ros/test_results/px4/rostest-launch_pub_test.xml
[Testcase: testpub_test] ... ok
```

**Comment by dagar** [comment]
I'm seeing similar failures in master that must have slipped through https://github.com/PX4/Firmware/pull/11917. @julianoes FYI

 - http://ci.px4.io:8080/blue/organizations/jenkins/PX4_misc%2FFirmware-SITL_tests/detail/master/868/pipeline
 - http://ci.px4.io:8080/blue/organizations/jenkins/PX4_misc%2FFirmware-SITL_tests_coverage/detail/master/277/pipeline

**Comment by TSC21** [comment]
I am also finding a weird behavior when building PX4 under a catkin ws and launching the PX4 SITL and Gazebo environment, where the EKF2 never gets initialized with valid GPS dara, resulting on not being able to change to takeoff or any auto/assisted modes. I can reproduce this issue in multiple different packages I have that are synced with upstream master. This was working before, so I am not sure what inteoduced this issue (this is verifiable before #11917 as well).

File: src/lib/systemlib/system_params.c [hunk 1]
```diff
@@ -100,8 +100,8 @@ PARAM_DEFINE_INT32(SYS_RESTART_TYPE, 2);
  *
  * Set the group of estimators used for multicopters and VTOLs
  *
- * @value 1 local_position_estimator, attitude_estimator_q
- * @value 2 ekf2
+ * @value 1 local_position_estimator, attitude_estimator_q (unsupported)
```
Comment (bkueng) [inline src/lib/systemlib/system_params.c]: Can we add a pure q estimator option for minimal setups?

Comment (jgoppert) [reply src/lib/systemlib/system_params.c]: I think as long as we could guarantee it wouldn't bit rot, that would be a good feature. Also for research/development I think stabilizing the interface for estimation (ekf2 module) and control (attitude_control /position_control module) into an API with swap-able code-generated back-ends would be nice. I have something along those lines that I hope to finish this summer at least for the estimation side.

Comment (dagar) [reply src/lib/systemlib/system_params.c]: @bkueng yes, let's followup in another PR.

**Pull request merged**